DUI3/Speckle.Connectors.DUI/ContainerRegistration.cs

```diff
@@ -40,10 +40,22 @@ public static void UseDUI(this IServiceProvider serviceProvider)
     {
       try
       {
-        serviceProvider
-          .GetRequiredService<ILoggerFactory>()
-          .CreateLogger("UnobservedTaskException")
-          .LogError(args.Exception, "Unobserved task exception");
+        var stackTrace = args.Exception.StackTrace;
+        if (stackTrace?.IndexOf("Speckle", StringComparison.InvariantCultureIgnoreCase) <= 0)
```

Comment:
In my testing. The `StackTrace` here appears to frequently be `null` on the `AggregateException` that is `args.Exception`
And instead, the inner exception has the real stack trace.

It looks like `.StackTrace` does not include the inner exception stack traces. I think this is important to ensure this logic is reliable.

Reply:
![image](https://github.com/user-attachments/assets/061d6260-83b9-4a16-9780-f8d8e835e75e)

Reply:
I didn't realize it was always an Aggregate exception.  I'm now handling it as such